frontend/packages/topology/src/utils/application-utils.ts

```diff
@@ -195,16 +193,14 @@ export const cleanUpWorkload = async (
   const resourceBuildConfigs = getBuildConfigsForResource(resource, resources);
   const isBuildConfigPresent = !_.isEmpty(resourceBuildConfigs);
 
-  const deleteModels = [ServiceModel, RouteModel];
+  const deleteModels = [ServiceModel, RouteModel, ImageStreamModel];
   const knativeDeleteModels = [KnativeServiceModel];
```

Comment:
You dropped below the `isImageStreamPresent` condition where `ImageStreamModel` was added to `deleteModels` and `knativeDeleteModels`. So I expect that it's required that you add this here also to the `knativeDeleteModels`, right?

Reply:
I missed adding the `ImageStreamModel` to the knativeDeleteModels. Thanks !